demisto_sdk/commands/common/tools.py

```diff
@@ -822,15 +822,15 @@ def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignor
         if 'brandName' in _dict and 'transformer' in _dict:
             return FileType.OLD_CLASSIFIER
 
-        if 'transformer' in _dict and 'keyTypeMap' in _dict:
-            return FileType.CLASSIFIER
+        if ('transformer' in _dict and 'keyTypeMap' in _dict) or 'mapping' in _dict:
+            if _dict.get('type') == 'classification':
+                return FileType.CLASSIFIER
+            else:
+                return FileType.MAPPER
```

Comment:
```suggestion
            elif _dict.get('type') and 'mapping' in _dict.get('type'):
                return FileType.MAPPER
            else:
               return None
```

might give better results - here is my thought:
 - if the type is `classification` it is a classifier.
 - if the type has `mapping` in it - it is a mapper.
 - if not both then we cannot identify the file type and we should get the "file not identified error" hence return None. 
We can also print a warning message here to direct the user to the type field. but we need to make sure it looks good.